src/admin-views/edit_admin_panel.php

```diff
@@ -0,0 +1,159 @@
+	?>
+	<?php if ( get_post_meta( $post_id, '_EventOrigin', true ) === 'community-events' ) {
+		?>
+		<?php // @TODO: this should get moved to Community Events? Use tribe_events_tickets_pre_edit ?>
+		<div>
+			<div class="tribe_sectionheader updated">
+				<p class="error-message"><?php esc_html_e( 'This event was created using Community Events. Are you sure you want to sell tickets for it?', 'event-tickets' ); ?></p>
+			</div>
+		</div>
+	<?php
+	}
+	?>
+
+	<div id="ticket_form" class="ticket_form tribe_sectionheader">
+		<div id="ticket_form_table" class="eventtable ticket_form">
+			<?php // @TODO: Do these need to get renamed for RSVPs? ?>
+			<h4 class="ticket_form_title_add"><?php esc_html_e( 'Add new ticket', 'event-tickets' ); ?></h4>
+			<h4 class="ticket_form_title_edit"><?php esc_html_e( 'Edit ticket', 'event-tickets' ); ?></h4>
+			<section id="ticket_form_main" class="main">
+				<div class="input_block">
+					<label class="ticket_form_label" for="ticket_name"><?php esc_html_e( 'Type:', 'event-tickets' ); ?></label>
+					<input type='text' id='ticket_name' name='ticket_name' class="ticket_field" size='25' value='' />
+				</div>
+				<?php // @TODO: should this get moved to ET+ ?>
+				<fieldset class="input_block">
+					<legend class="ticket_form_label"><?php esc_html_e( 'Sell using:', 'event-tickets' ); ?></legend>
+					<?php
+					$checked = true;
+					foreach ( $modules as $class => $module ) {
+						?>
+						<input <?php checked( $checked ); ?> type="radio" name="ticket_provider"
+															id="<?php echo esc_attr( $class . '_radio' ); ?>"
+															value="<?php echo esc_attr( $class ); ?>"
+															class="ticket_field ticket_provider"
+															tabindex="-1">
+						<span><?php echo esc_html( apply_filters( 'tribe_events_tickets_module_name', $module ) ); ?></span>
+						<?php
+						$checked = false;
+					}
+					?>
+				</fieldset>
+				<?php
+				/**
+				 * Allows for the insertion of additional content into the ticket edit form - main section
+				 *
+				 * @var Post ID
+				 * @var null Ticket ID
+				 */
+				do_action( 'tribe_events_tickets_metabox_edit_main', $post_id, null ); ?>
+			</section>
+			<div class="accordion">
+				<button class="accordion-header" type="button">
+					<?php esc_html_e( 'Advanced', 'event-tickets' ); ?>
+				</button>
+				<section id="ticket_form_advanced" class="advanced accordion-content">
+					<h4 class="accordion-label"><?php esc_html_e( 'Advanced Settings', 'event-tickets' ); ?></h4>
+					<div class="input_block">
+						<label class="ticket_form_label" for="ticket_description"><?php esc_html_e( 'Ticket Description:', 'event-tickets' ); ?></label>
+						<textarea rows="5" cols="40" name="ticket_description" class="ticket_field"
+									id="ticket_description"></textarea>
+						<div class="input_block">
+							<label><input type="checkbox" name="tribe_show_description" value="1"> Show description on front end and emailed tickets.</label>
```

Comment:
Probably not essential - was just checking in case it had been missed. We should make the text translatable, though.